ccd-adapter/src/main/java/uk/gov/hmcts/cmc/ccd/domain/CCDScannedDocumentType.java

```diff
@@ -0,0 +1,8 @@
+package uk.gov.hmcts.cmc.ccd.domain;
+
+public enum CCDScannedDocumentType {
+    cherished,
+    other,
+    form,
+    coversheet
```

Comment:
should these be in caps?

Reply:
this is how the Bulk Scan sends entry to CCD, so we cant change it.